src/components/backend-ai-scaling-group-list.ts

```diff
@@ -438,7 +617,10 @@ export default class BackendAIScalingGroupList extends BackendAIPage {
               id="add-scaling-group"
               icon="add"
               label="${_t('button.Add')}"
-              @click=${() => this._launchDialogById('#create-scaling-group-dialog')}>
+              @click=${() => {
+    this._clearValues();
```

Comment:
Any reasons for using `_clearValues()`?
If you want to restore the values to the previous value, then we should change the name of functions to more proper terms.

Reply:
first reason is reset scheduler options's value to default when user open 'create dialog', 
second reason is allow to user reset all value to default when user want default value.
and i changed name '_clearValues' to '_resetAllValues'